tests/trainer/flags/test_logger.py

```diff
@@ -0,0 +1,36 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
what's the difference between this file and the one here? https://github.com/PyTorchLightning/pytorch-lightning/blob/master/tests/trainer/properties/test_loggers.py

let's merge these together given they test the same things.

Reply:
Technically one tests the Trainer properties and one tests the Trainer flags (constructor args). I can merge them if you want but they are technically different things.

Reply:
this is also asserting on the trainer properties to confirm the flags passed in work as expected. i think they're basically the same thing

Reply:
you are totally right, sorry. updated.